opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -48,47 +49,27 @@ def __init__(
         self,
         value_type: Type[metrics_api.ValueT],
         enabled: bool,
-        aggregator: Aggregator,
+        labels: Tuple[Tuple[str, str]],
+        metric: metrics_api.MetricT,
     ):
-        self.value_type = value_type
-        self.enabled = enabled
-        self.aggregator = aggregator
-        self._ref_count = 0
-        self._ref_count_lock = threading.Lock()
+        self._value_type = value_type
+        self._enabled = enabled
+        self._labels = labels
+        self._metric = metric
 
     def _validate_update(self, value: metrics_api.ValueT) -> bool:
-        if not self.enabled:
+        if not self._enabled:
             return False
-        if not isinstance(value, self.value_type):
+        if not isinstance(value, self._value_type):
             logger.warning(
-                "Invalid value passed for %s.", self.value_type.__name__
+                "Invalid value passed for %s.", self._value_type.__name__
             )
             return False
         return True
 
     def update(self, value: metrics_api.ValueT):
-        self.aggregator.update(value)
-
-    def release(self):
```

Comment:
Even after this PR, bound instruments still use resources indefinitely, right?

Reply:
Release is still needed, when user doesn't use bound instrument to call update? I assume SDK creates and releases a boundinstrument temporarily in this case?

Reply:
The spec does not enforce the releasing of bound instruments. As @c24t pointed out: "The fact that we MAY waste resources doesn't necessarily mean that we SHOULD. We may want to revisit this once views are working.". Removing them for now greatly simplifies the views work, so i'm removing them for now and we might revisit in the future.